articles/getting-started/the-basics.md

```diff
@@ -5,93 +5,91 @@ toc: true
-- The cloud managed by Auth0
-- A cloud managed by you
-- An [on-premise installation](/appliance)
-:::
+If you need to use your own domain (for example, `myapp-staging.com`), you can configure a custom domain. To read more about this non-free beta feature, see [Custom Domains](/custom-domains).
 
-## Client
+## Clients
 
-Now that you have an account, we need to know about your app(s) that will be using our services. To that end, you must register each application. We use the term **client** to refer to an application (like [OAuth 2.0 does](https://tools.ietf.org/html/rfc6749#page-6)).
+Like OAuth 2.0, Auth0 calls the applications it serves **clients**. Think of clients as your applications' Auth0 credentials; every app needs its own client before it can talk to Auth0.
 
-When you create a client in the [Dashboard](${manage_url}/#/clients), the first piece of information we ask for is its type. This can be one of the following.
+When creating a client in the [Dashboard](${manage_url}/#/clients), you must choose its type:
 
 ![Client Types](/media/articles/getting-started/client-types.png)
 
-Each client is assigned a **Client ID** upon creation. This is an alphanumeric string and it's the unique identifier for your client (such as `q8fij2iug0CmgPLfTfG1tZGdTQyGaTUA`). It cannot be modified and you will be using it in your application's code when you call Auth0 APIs.
+Each client is assigned a **Client ID** upon creation. This immutable alphanumeric string is your app's username. The **Client Secret** is its password, so of course, **you must not share it with others**. Anyone that learns the Client Secret can impersonate your application.
 
-Another important piece of information is the **Client Secret**. Think of it as your client's password which **must be kept confidential at all times**. If anyone gains access to your Client Secret they can impersonate your application and access protected resources.
+See [Clients](/clients) to learn more.
 
-In our example, `ExampleCo` has two apps: a web app (running on a server) and a mobile app. Hence, they would create two clients: one of type `Regular Web Applications`, and one of type `Native`.
+## Connections
 
-::: note
-We won't get into details on how to create a client, since this article is meant to provide a high level overview. However, if you want to know more, refer to the [Clients](/clients) documentation.
-:::
+Just as clients help your applications connect to Auth0, **connections** help Auth0 connect to the identity providers that authenticate your applications' users. Auth0 supports a wide variety of identity providers and credential types:
 
-## Connection
+- [Database connections](/connections/database): Usernames and passwords stored in either your Auth0 account or your own (external) database.
+- [Social logins](/identityproviders#social): Google, Facebook, Twitter, and more.
```

Comment:
I haven't reviewed the rest of this, just spun up the Heroku build and saw that fail - my guess is that the issue is that these items need to be `/connections/social` and `/connections/enterprise` - traversing the document structure rather than the URLs they end up being.